src/sage/groups/matrix_gps/coxeter_group.py

```diff
@@ -557,13 +569,14 @@ def _positive_roots_reflections(self):
         simple_roots = FreeModule(self.base_ring(), self.ngens()).gens()
 
         refls = self.simple_reflections()
+        refls_index = {refl_i[1]: refl_i[0] for refl_i in enumerate(refls)}
```

Comment:
```suggestion
        refls_index = {refl_i[1]: refl_i[0] for refl_i in enumerate(refls.keys())}
```
Sorry, I forgot this was a `Family` and not a `dict`.